spec/definitions/CustomerInformation.yaml

```diff
@@ -0,0 +1,22 @@
+type: object
+properties:
+  lifetimeMetrics:
```

Comment:
I find this kind of strange.  The description of the endpoint is that it's lifetime summary metrics.  So why do we need a lifetimeMetrics object?